### ISSUE TITLE
Unable to build with scons and latest winword commit in nvda remote

### ISSUE THREAD

**Issue by nromey** [nvaccess/nvda#6391]
When compiling with scons and an updated master, I'm unable to compile due to not being able to find a boost library.  The error occurs with fields.h in nvda remote and the error stack is below.

C:\WINDOWS\system32\cmd.exe
Microsoft Windows [Version 10.0.14393]                                                                                  
(c) 2016 Microsoft Corporation. All rights reserved.                                                                    
                                                                                                                        
C:\Users\ner>cd c:\nvdasource\nvda                                                                                      
                                                                                                                        
c:\nvdasource\nvda>scons source                                                                                         
scons: Reading SConscript files ...                                                                                     
scons: done reading SConscript files.                                                                                   
scons: Building targets ...                                                                                             
interfaceAction(["source\comInterfaces\ISimpleDOM.py"], ["source\typelibs\ISimpleDOMNode.tlb"])                         
interfaceAction(["source\comInterfaces\mathPlayer.py"], ["source\typelibs\mathPlayerDLL.tlb"])                          
cl /Fobuild\x86\remote\winword.obj /c build\x86\remote\winword.cpp /TP /EHsc /nologo /wd4091 /arch:IA32 /W3 /WX /Od /MT 
/D_USING_V110_SDK71_ /DUNICODE /D_CRT_SECURE_NO_DEPRECATE /DLOGLEVEL=15 /D_WIN32_WINNT=_WIN32_WINNT_WS03 /DNDEBUG /Iincl
ude /ImiscDeps\include /Ibuild\x86 /Z7                                                                                  
winword.cpp                                                                                                             
C:\nvdasource\nvda\build\x86\remote/WinWord/Fields.h(19): fatal error C1083: Cannot open include file: 'boost/optional.h
pp': No such file or directory                                                                                          
scons: *** [build\x86\remote\winword.obj] Error 2                                                                       
scons: building terminated because of errors.                                                                           
                                                                                                                        
c:\nvdasource\nvda>                                                                                                     
                                                                                                                        
                                                                         
I tried installing boost and bootstrapping it but that didn't make a difference as boost isn't included or at least I don't see it as beling included.
Since I'm working off my own branch, I wanted to compile a launcher myself. Scons source and scons launcher throw the error above.

**Comment by josephsl** [comment]
CC @feerrenrut, I can confirm this. Thanks.

**Comment by jcsteh** [comment]
Your miscDeps submodule is out of date. Run:
git submodule sync
git submodule update

**Comment by josephsl** [comment]
Hi, Got it. I’m afraid Boost dependency is not listed under “installed packages” in readme. Thanks.

**Comment by jcsteh** [comment]
It doesn't need to be because it's part of miscDeps; you don't need to 
install it. However, it should probably be listed as being in a 
submodule just for reference purposes. @feerrenrut, can you update the 
"Git Submodules" section of readme.md to list Boost optional?

**Comment by feerrenrut** [comment]
Yep, no problem @jcsteh. I have added this now.
